datapackages/2018-19/epre/datapackage.json

```
@@ -4,13 +4,13 @@
     "profile": "tabular-data-package",
     "resources": [
         {
-            "path": "https://s3-eu-west-1.amazonaws.com/manual-uploads.vulekamali.gov.za/treasury-auto-checked/estimates-of-provincial-expenditure-south-africa-2018-19.csv",
+            "path": "https://s3-eu-west-1.amazonaws.com/manual-uploads.vulekamali.gov.za/treasury-auto-checked/estimates-of-provincial-expenditure-south-africa-2018-19-consistent-column-names.csv",
             "profile": "tabular-data-resource",
             "name": "provincial-2018-19",
             "format": "csv",
             "mediatype": "text/csv",
             "encoding": "utf-8",
-            "schema": "https://raw.githubusercontent.com/vulekamali/data-checks/master/schema/epre.json"
+            "schema": "https://raw.githubusercontent.com/vulekamali/data-checks/fluenty-delena-2017-18-epre/schema/epre.json"
```

revert to master before merging once tests pass
```suggestion
            "schema": "https://raw.githubusercontent.com/vulekamali/data-checks/master/schema/epre.json"
```